tensorflow/python/keras/applications/mobilenet_v2.py

```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
     ValueError: in case of invalid argument for `weights`,
       or invalid input shape or invalid alpha, rows when
       weights='imagenet'
+      
+  Usage Example:
+  >>> from tensorflow.keras.applications import MobileNetV2
+  >>> IMG_SHAPE = (224, 224, 3)
+  >>> model = MobileNetV2(input_shape =IMG_SHAPE , include_top = True)
```

Comment:
`input_shape=IMG_SHAPE` is the pylint style, same for `include_top=True`.

Reply:
what should be here instead?

Reply:
No need for spaces around `=` in function parameters. Make sure you run pylint on the code and fix any errors